src/Tile/_ProductTileContent.js

```diff
@@ -1,37 +1,13 @@
-import classnames from 'classnames';
-import CustomPropTypes from '../utils/CustomPropTypes/CustomPropTypes';
-import PropTypes from 'prop-types';
 import React from 'react';
+import TileContent from './_TileContent';
 
 const ProductTileContent = props => {
-    const { title, children, className, headingLevel, titleProps, ...rest } = props;
+    const { title } = props;
 
-    const tileProductContentClasses = classnames(
-        'fd-product-tile__content',
-        className
-    );
-
-    const HeadingTag = `h${headingLevel}`;
-
-    return (
-        <div {...rest} className={tileProductContentClasses}>
-            <HeadingTag {...titleProps} className='fd-product-tile__title'>{title}</HeadingTag>
-            {children}
-        </div>
-    );
+    return (<TileContent productTile title={title}
+        {...props} />);
```

Comment:
Can we just pass props without pruning title here?

Reply:
Yup this was my bad, I was testing this out when I was having issues with a warning message and forgot to remove.